block/manager_test.go

```diff
@@ -71,20 +70,16 @@ func TestInitialState(t *testing.T) {
 
 	key, _, _ := crypto.GenerateEd25519Key(rand.Reader)
 	conf := config.BlockManagerConfig{
-		BlockTime:   10 * time.Second,
-		NamespaceID: types.NamespaceID{1, 2, 3, 4, 5, 6, 7, 8},
+		BlockTime: 10 * time.Second,
 	}
 
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
 			assert := assert.New(t)
 			logger := test.NewFileLoggerCustom(t, test.TempLogFileName(t, c.name))
-			dalc := getMockDALC(logger)
-			defer func() {
-				require.NoError(t, dalc.Stop())
-			}()
+			dalc := &da.DAClient{DA: goDATest.NewDummyDA(), Logger: logger}
 			agg, err := NewManager(key, conf, c.genesis, c.store, nil, nil, dalc, nil, logger, nil)
-			assert.NoError(err)
+			require.NoError(t, err)
```

Comment:
I suggest sticking with `assert` here so that other test cases still run. If we use `require`, the test fails as soon as the first `require` statement fails, but not the case with `assert`